src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -118,22 +118,23 @@
                     ItemClick="ItemsList_ItemClick"
                     ItemTemplate="{StaticResource ListItemViewModelTemplate}"
                     ItemsSource="{x:Bind ViewModel.FilteredItems, Mode=OneWay}"
+                    RightTapped="ItemsList_RightTapped"
```

Comment:
Just to 3x check, this will cover all use cases, right? Right-click with mouse, long-press with touch?

Reply:
Indeed